### PR TITLE
Improving design of epg model

### PR DESCRIPTION
**Pull request by miloit**
Improving design of epg model by adding text color option

### CODE REVIEW

**Review by martonborzak — COMMENTED** [review]
Could you explain what does this change do and how one can use it?

**Comment by miloit** [comment]
> Could you explain what does this change do and how one can use it?

You can simply change also the color of the text of the EPG model

File: components/media_player/ui/CardMediaPlayerEPGView.qml
Comment (zehnm) [inline components/media_player/ui/CardMediaPlayerEPGView.qml:155]: Where is `item_epgItemTextColor` defined? I can't find it.

Comment (miloit) [reply components/media_player/ui/CardMediaPlayerEPGView.qml:155]: in the integrations.ibrary epgmodel header file

    `roles[EpgItemTextColorRole] = "item_epgItemTextColor";`

Comment (zehnm) [reply components/media_player/ui/CardMediaPlayerEPGView.qml:155]: @martonborzak could you check this? To me as a non QML / JS developer I don't see the magic how this could work.
Nevertheless, if it's defined in the other integrations.library pull request, then we have to wait until it is merged & released, then update the dependency in this PR.

Comment (martonborzak) [reply components/media_player/ui/CardMediaPlayerEPGView.qml:155]: It's defined in the QAbstractListmodel `roles[EpgItemColorRole] = "item_epgItemColor";`, so it should work, once the integrations.library is merged.

Comment (zehnm) [reply components/media_player/ui/CardMediaPlayerEPGView.qml:155]: That still doesn't answer my ignorance of QML since I only see a string of the same name but no property / object, but I trust you :-)

@martonborzak could you check the related PR YIO-Remote/integrations.library#15 ? Then I'll merge and create new releases.

Comment (martonborzak) [reply components/media_player/ui/CardMediaPlayerEPGView.qml:155]: It's good.

The name of string is coming from how the listview handles the QAbstracListmodel. When you pass the model to the listview, it know all the properties automatically that was defined in c++.

**Review by zehnm — APPROVED** [review]

**Pull request merged**
